src/app/layout.tsx

```diff
@@ -16,14 +19,43 @@ export default function RootLayout({
 }: {
   children: React.ReactNode;
 }) {
-  return (
-    <html lang="en">
-      <body className={inter.className}>
-        <ExtensionProvider>
+  const providerType = process.env.PROVIDER_TYPE;
+  const apiType = process.env.API_TYPE;
```

Comment:
Let's add .env.example, and provide instructions to README how to set-up them and other stuff